jwk.go

```diff
@@ -0,0 +1,57 @@
+package clerk
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+
+	"github.com/go-jose/go-jose/v3"
+)
+
+type JSONWebKeySet struct {
+	APIResource
+	Keys []JSONWebKey `json:"keys"`
+}
+
+type JSONWebKey struct {
+	APIResource
+	jose.JSONWebKey
```

Comment:
🔧 Since we don't want to expose jose, shouldn't we avoid embedding this and instead wrap it with our own implementation?

Reply:
Well, yes! Great catch, I missed this. Used an unexported field instead.